libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -209,9 +239,17 @@ namespace SohImGui {
 
     void ImGuiProcessEvent(EventImpl event) {
         switch (impl.backend) {
+#ifdef __WIIU__
+        case Backend::GX2:
+            if (!ImGui_ImplWiiU_ProcessInput((ImGui_ImplWiiU_ControllerInput*)event.gx2.input)) {
```

Comment:
Oh yeah, this used to disable game input while the virtual keyboard is opened. Since game input is now always disabled if the ImGui menu is open, this is no longer necessary.